assets/stylesheets/_components.scss

```diff
@@ -207,7 +207,6 @@
 @import 'my-sites/ads/style';
 @import 'my-sites/all-sites/style';
 @import 'my-sites/all-sites-icon/style';
-@import 'my-sites/category-selector/style';
```

Comment:
We also need to remove this `_z-index.scss` entry:

https://github.com/Automattic/wp-calypso/blob/0a08539/assets/stylesheets/shared/functions/_z-index.scss#L128